index.ts

```diff
@@ -132,7 +136,10 @@ function getTSConfig(options: Options, fileName: string): Options {
 		options.rootDir = configParseResult.options.rootDir;
 	}
 	options.files = configParseResult.fileNames;
-	return options;
```

Comment:
To make sure I'm not missing anything, this is being removed because we're mutating the options object that was passed in? Yeah, definitely not what we want to do, so I see your point with the TODO.